cookbook/zh/agent_app.md

```diff
@@ -513,3 +513,8 @@ await app.deploy(LocalDeployManager(host="0.0.0.0", port=8091))
 ```
 
 更多部署选项和详细说明，请参考 {doc}`advanced_deployment` 文档。
+
+AgentScope Runtime 提供了Serverless的部署方案，您可以将您的Agent部署到 ModelStudio(FC) 或 AgentRun 上。
+查看 [ModelStudio部署](https://runtime.agentscope.io/en/advanced_deployment.html#method-4-modelstudio-deployment)
+和 [AgentRun部署](https://runtime.agentscope.io/en/advanced_deployment.html#method-5-agentrun-deployment)
```

Comment:
In Jupyter Book, it is recommended to use the `{doc}` syntax for cross-references. This ensures that links are properly resolved during the build process and remain consistent across pages.
Suggested change:

```
{doc}`advanced_deployment`  
```

Please update the links to follow this format so they are compatible with Jupyter Book and maintain link integrity.